With Flexbox/css/index.css

```diff
@@ -0,0 +1,69 @@
+body {
+    font-family: 'Titillium Web', sans-serif;
```

Comment:
Here I would add the following flex rules to center the `.main-container` on the page

```
    display: flex;
    flex-direction: column;
    align-items: center;
```